### ISSUE TITLE
"Add it" button does not show up when list or map is matched

### ISSUE THREAD

**Issue by betsydupuis** [public-accountability/littlesis-rails#1731]
<img width="741" alt="Screenshot 2024-08-20 at 6 19 30 PM" src="https://github.com/user-attachments/assets/b6a4b551-45c6-45b5-8ae2-63221a10cadc">

**Comment by josephlacey** [comment]
Hi @betsydupuis, what's the context for this search?  What are you adding the list or the map to?

**Comment by betsydupuis** [comment]
> what's the context for this search? What are you adding the list or the map to?

Hi @josephlacey I was searching for that name in the regular search and it did not exist, but it existed in the context of a map.

If I search for a completely nonexistent string, then the "add it" button appears. 

I haven't added "Foundation for Freedom Online" so you can test it if you want. 

I think maps are fuzzy matching and entities are exact matches. This happens with lists, but I haven't been able to replicate it.

I think there's a UX case for removing the logic that hides and shows the button and always showing it. It's always possible that you get search results, but the entity you want isn't there.

 **Non-existant string**
<img width="923" alt="Screenshot 2024-08-20 at 6 21 04 PM" src="https://github.com/user-attachments/assets/151bc46e-5ead-4fb9-ae94-508106b314d2">

 **Fuzzy string match to map and list**
<img width="982" alt="Screenshot 2024-08-20 at 6 31 21 PM" src="https://github.com/user-attachments/assets/5a3cb1f1-8013-4570-a53b-108a67f83813">

**Comment by jessib** [comment]
@josephlacey : to confirm, is the desired functionality to always show the "add it" button? That seems reasonable to me, but want to confirm. Thanks!

**Comment by josephlacey** [comment]
> [@josephlacey](https://github.com/josephlacey) : to confirm, is the desired functionality to always show the "add it" button? That seems reasonable to me, but want to confirm. Thanks!

@jessib Yeah, in case what you're looking for has potential matches but isn't quite right.
